src/Interfaces/LazySet.jl

```diff
@@ -682,3 +683,34 @@ end
 
 isoperationtype(::LazySet) = error("`isoperationtype` cannot be applied to " *
                                    "a set instance; use `isoperation` instead")
+
+"""
+    isequivalent(X::LazySet, Y::LazySet)
+
+Return whether two LazySets are equal in the mathematical sense, i.e. equivalent.
+
+### Input
+
+- `X` -- any `LazySet`
+- `Y` -- another `LazySet`
+
+### Output
+`true` iff `X` is equivalent to `Y`.
+
+### Examples
+
+```
+julia> X = BallInf([0.1, 0.2], 0.3)
+julia> Y = convert(HPolytope, X)
+julia> X == Y
+false
+julia> is_equivalent(X, Y)
+true
+```
+"""
+function isequivalent(X::LazySet, Y::LazySet)
+    if X == Y
```

Comment:
Ok; this check is useful if the sets are of the same type, that was the case?

Reply:
Yes, I added this because it is not expensive to check `X == Y` and could speed up the check if `X == Y` is already true.

Reply:
Ok, makes sense.